<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogicGalleryWidget</class>
 <widget class="QDialog" name="LogicGalleryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logic Gallery</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="LayoutMain">
    <item>
     <layout class="QHBoxLayout" name="LayoutGallery">
      <item>
       <layout class="QVBoxLayout" name="LayoutSelection">
        <item>
         <spacer name="VSpacer1">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QRadioButton" name="RBTouch">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Touch ON/OFF</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RBCalibration">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Calibration</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RBTaskSwitch">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Task Switch</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="VSpacer2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QRadioButton" name="RBTouchGPIO">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Touch ON/OFF with GPIO</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RBCalibrationGPIO">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Calibration with GPIO</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RBTaskSwitchGPIO">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Task Switch with GPIO</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="VSpacer3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="LayoutGPIO">
          <item>
           <widget class="QLabel" name="TextGPIO">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>GPIO #</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="CBGPIO">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="Gallery" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="LayoutButtons">
      <item>
       <spacer name="HSpacer1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="BtnInsert">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Insert</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnCancel">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
